rocksdb/rocksdb.nim

```diff
@@ -465,6 +465,22 @@ proc releaseSnapshot*(db: RocksDbRef, snapshot: SnapshotRef) =
     rocksdb_release_snapshot(db.cPtr, snapshot.cPtr)
     snapshot.setClosed()
 
+proc flush*(db: RocksDbRef, cfs: openArray[ColFamilyHandleRef]): RocksDBResult[void] =
```

Comment:
@arnetheduck Is the goal to simply to flush all the opened column families? If so, I'll remove the cfs parameter and just flush all the column familes in `ColFamilyTableRef`

Reply:
The Java library has 3 alternatives: https://javadoc.io/doc/org.rocksdb/rocksdbjni/9.6.1/org/rocksdb/RocksDB.html#flush-org.rocksdb.FlushOptions-

I can do something similar if we want to have more options. One of the functions has no cf parameter which implies flushing all opened column families.

Reply:
Yes actually, you are correct. That one is using the default CF, my mistake. I'll follow the pattern and implement as you suggested